src/domain/users/index.types.d.ts

```diff
@@ -30,6 +33,30 @@ type UserQuizQuestion = {
   completed: boolean
 }
 
+// TODO: move to camelCase base // migration needed
+// type PhoneMetadata = {
+//   carrier: {
+//     errorCode: string | undefined // check this is the right syntax
+//     mobileCountryCode: string | undefined
+//     mobileNetworkCode: string | undefined
+//     name: string | undefined
+//     type: "landline" | "voip" | "mobile"
+//   }
+//   countryCode: string | undefined
+// }
+
+type PhoneMetadata = {
+  // from twilio
+  carrier: {
+    error_code: string // check this is the right syntax
+    mobile_country_code: string
+    mobile_network_code: string
+    name: string
+    type: "landline" | "voip" | "mobile"
```

Comment:
For this kind of type that has only certain allowed values we've been defining a `const Object` and making a type out of it. Look at the `LedgerTransactionType` for example:
[Type](https://github.com/GaloyMoney/galoy/blob/main/src/domain/ledger/index.types.d.ts#L16-L17) [object](https://github.com/GaloyMoney/galoy/blob/main/src/domain/ledger/index.ts#L10-L26). With this method you can then reference the values like `PhoneType.Landline`.